feat(rdb): apply maintenance MTA-4494 (#3088)
scaleway/docs-content: managed-databases/postgresql-and-mysql/how-to/apply-maintenance.mdx

---
meta:
  title: How to apply scheduled maintenance to a Database Instance
  description: This page explains how to apply maintenance to a Database Instance
content:
  h1: How to apply scheduled maintenance to a Database Instance
  paragraph: This page explains how to apply maintenance to a Database Instance
tags: managed-database postgresql mysql database-instance maintenance
dates:
  validation: 2024-04-19
  posted: 2024-04-19
categories:
  - managed-databases
  - postgresql-and-mysql
---

From time to time your Scaleway Managed Databases have to undergo maintenance to make sure that your nodes are up-to-date and have all the tools necessary to maintain a healthy lifecycle. For example, your engine version might need to be upgraded to the latest available minor version, or certain patches might need to be implemented.

> These maintenance are set up, run, and scheduled by Scaleway, but you can select when to apply them to avoid interruptions during peak times.

Reviewer: ```suggestion
This maintenance is set up, run, and scheduled by Scaleway, but you can select when to apply it to avoid interruptions during peak times.
```

Author: ```suggestion
These maintenance operations are set up, run, and scheduled by Scaleway, but you can select when to apply them to avoid interruptions during peak times.
```

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart) scheduled for maintenance

1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Look for an orange info icon to identify which Database Instances are scheduled for maintenance.
3. Click the Database Instance name or <Icon name="more" /> > **More info** to access the database information page.

    If your Database Instance is scheduled for maintenance, you will see a banner on the top of the page informing you when the scheduled maintenance will take place. You can either wait for the maintenance to be applied automatically at the expected date and time, or you can apply it immediately.

    <Message type="important">

>       Certain maintenance cannot be applied manually. In this case, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.

Reviewer 2: ```suggestion
      Certain maintenances cannot be applied manually. In this case, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.
```

>       Certain maintenance cannot be applied manually. In this case, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.

Reviewer 3: we don't put examples here? (like hot security patches for example)

Author: ```suggestion
      Certain maintenance cannot be applied manually, such as hot security patches. In these cases, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.
```

>       Certain maintenance cannot be applied manually, such as hot security patches. In these cases, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.

Reviewer: ```suggestion
      Certain types of maintenance cannot be applied manually, such as hot security patches. In these cases, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.
```

    </Message>

>     <Lightbox src="scaleway-apply-db-maintenance.webp" alt="The banner says: 'Maintenance is scheduled on May 17, 2024 at 02:00 PM (UTC). A maintenance will be applied on your instance. The database will be unavailable for a few minutes. You can apply this maintenance now or wait for it to be automatically applied.' It includes a link called 'see status' where the user can check for more details, and a button labeled 'Apply maintenance now' they can click to apply the maintenance immediately" />

Reviewer: I know this is incidental if it's in the console but "a maintenance" is not correct - it is an uncountable noun

Author: ```suggestion
    <Lightbox src="scaleway-apply-db-maintenance.webp" alt="The banner says: 'Maintenance is scheduled on May 17, 2024 at 02:00 PM (UTC). A maintenance operation will be applied to your instance. The database will be unavailable for a few minutes. You can apply this maintenance now or wait for it to be automatically applied.' It includes a link called 'see status' where the user can check for more details, and a button labeled 'Apply maintenance now' they can click to apply the maintenance immediately" />
```

Author: ```suggestion
```
